detect-nuget-inspector/detect-nuget-inspector/Inspection/Inspectors/ProjectInspector.cs

```diff
@@ -204,6 +203,16 @@ public Container GetContainer()
                     }
                 }
 
+                Options.ProjectAssetsJsonPath = GetProjectAssetsJsonPathFromNugetProperty(Options.ProjectDirectory, Options.ProjectName, Options.ProjectAssetsJsonPath);
+                if (!String.IsNullOrWhiteSpace(Options.ProjectAssetsJsonPath) && File.Exists(Options.ProjectAssetsJsonPath))
+                {
+                    Console.WriteLine("Using assets json file configured in property file: " + Options.ProjectAssetsJsonPath);
```

Comment:
Is this accurate? Couldn't `Options.ProjectAssetsJsonPath` be the one set in line 61 at this point?

Reply:
Good question. There are cases where line 61 is alone set, line 206 alone is set and both lines are set.

Reply:
Discussed with @andrian-synopsys and pushed improved changes.